### ISSUE TITLE
[BUG] Intermittent failure since 3.19.0 on PAVICS, even without caching enabled

### ISSUE THREAD

**Issue by tlvu** [Ouranosinc/Magpie#492]
**Describe the bug**
We observed the following intermittent errors after going-live PR https://github.com/bird-house/birdhouse-deploy/pull/224 on prod, without caching enabled.

```
10:46:02  _____________________ notebooks/stress-tests.ipynb::Cell 3 _____________________
10:46:02  Notebook cell execution failed
10:46:02  Cell 3: Cell execution caused an exception
10:46:02  
10:46:02  Input:
10:46:02  # NBVAL_IGNORE_OUTPUT
10:46:02  
10:46:02  # Executing WPS test on multiple threads
10:46:02  wps_urls = [
10:46:02      f"{TWITCHER_URL}/{bird}/wps?service=wps&request=getcapabilities"
10:46:02      for bird in TEST_WPS_BIRDS
10:46:02  ]
10:46:02  
10:46:02  for execution, url in enumerate(wps_urls):
10:46:02      run_threads(target_test = stress_test,
10:46:02                  url = url,
10:46:02                  execution_id = execution,
10:46:02                  n_threads = TEST_N_THREADS_WPS,
10:46:02                  runs_per_threads = TEST_RUNS_WPS)
10:46:02  
10:46:02  Traceback:
10:46:02  
10:46:02  ---------------------------------------------------------------------------
10:46:02  AssertionError                            Traceback (most recent call last)
10:46:02  /tmp/ipykernel_646/3472822828.py in <module>
10:46:02       12                 execution_id = execution,
10:46:02       13                 n_threads = TEST_N_THREADS_WPS,
10:46:02  ---> 14                 runs_per_threads = TEST_RUNS_WPS)
10:46:02  
10:46:02  /tmp/ipykernel_646/1081959450.py in run_threads(target_test, url, execution_id, n_threads, runs_per_threads)
10:46:02       37     assert (
10:46:02       38         stress_test_progression.failed_threads_count == 0
10:46:02  ---> 39     ), stress_test_progression.fail_report()
10:46:02       40 
10:46:02       41     print(f"All threads passed {target_test.__name__} function.")
10:46:02  
10:46:02  AssertionError: This test failed due to bad code received from the requests.
10:46:02   Number of failed thread(s) : 1/ 3
10:46:02   Number of failed requests : 1/300
10:46:02  
10:46:02  
10:46:02   Failed Test
10:46:02   Stress Test:
10:46:02      Test:
10:46:02          code: 200
10:46:02          runs: 100
10:46:02          max-avg-time:  1s
10:46:02          max-err-code:  0
10:46:02          sum-err-code:  1
10:46:02          timeout-abort: 0s
10:46:02          timeout-retry: 0
10:46:02          timeout-count: 0
10:46:02      Request:
10:46:02          method: GET
10:46:02          url:    https://pavics.ouranos.ca/twitcher/ows/proxy/hummingbird/wps?service=wps&request=getcapabilities
10:46:02          args:   {}
10:46:02      Times:
10:46:02          min: 0.122s
10:46:02          avg: 0.001s
10:46:02          max: 0.122s
10:46:02      Results:
10:46:02          Run                 Codes                 Delta                 Times            Timestamps
10:46:02            1               (!) 401                0.035s                0.122s   2021-12-07 15:44:40
10:46:02      Summary:
10:46:02          Detected 1 erroneous HTTP codes not equal to expected 200.
10:46:02          Test failed (status=-1).
10:46:02  
10:46:02   === Exceptions ===
10:46:02  
10:46:02   1. 401 Unauthorized
10:46:02   <?xml version="1.0" encoding="utf-8"?>
10:46:02  <ExceptionReport version="1.0.0"
10:46:02      xmlns="http://www.opengis.net/ows/1.1"
10:46:02      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
10:46:02      xsi:schemaLocation="http://www.opengis.net/ows/1.1 http://schemas.opengis.net/ows/1.1.0/owsExceptionReport.xsd">
10:46:02      <Exception exceptionCode="NoApplicableCode" locator="AccessForbidden">
10:46:02          <ExceptionText>Access to service is forbidden.</ExceptionText>
10:46:02      </Exception>
10:46:02  </ExceptionReport>
```

`docker logs twitcher`:

I got many of those, no other stack-trace:
```
2021-12-07 15:15:42,471 WARNI [TWITCHER:172][ThreadPoolExecutor-0_1] Security check failed but was not handled as expected by 'is_verified' method.
Traceback (most recent call last):
  File "/opt/birdhouse/src/twitcher/twitcher/owsproxy.py", line 169, in owsproxy_view
    raise OWSAccessForbidden("Access to service is forbidden.")

````

It looks like as soon as the error started to happen, it happens more regularly.  See screenshot of consecutive failure after the first failure in prod.

![Screenshot from 2021-12-07 13-10-44](https://user-images.githubusercontent.com/11966697/145085692-0c6a5c87-16ab-4d2c-8d4f-1e4152691140.png)

**Comment by fmigneault** [comment]
resolved by #491

**Comment by fmigneault** [comment]
resolved by #491
